templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -175,6 +185,15 @@ objects:
     password: ${BROKER_PASS}
 
 - apiVersion: v1
+  kind: Secret
+  metadata:
+    name: registry-auth-secret
+    namespace: ansible-service-broker
+  data:
+    username: ${DOCKERHUB_USER}
+    password: ${DOCKERHUB_PASS}
+
+- apiVersion: v1
```

Comment:
The secret is still ok, but instead of passing it as a parameter on the cli to the broker, it should be in the registry section of the config.